.env

```
@@ -109,4 +109,6 @@ LLM_SUMMERIZATION=true
 # PUBLIC_APP_COLOR=yellow
 # PUBLIC_APP_DESCRIPTION="Making the community's best AI chat models available to everyone."
 # PUBLIC_APP_DATA_SHARING=1
-# PUBLIC_APP_DISCLAIMER=1
\ No newline at end of file
+# PUBLIC_APP_DISCLAIMER=1
+
+DISABLE_ASSISTANTS=false # set to true to disable assistants feature
```

should be disabled by default?
```suggestion
ENABLE_ASSISTANTS=false # set to true to enable this feature
```

---

The idea to make it enabled by default was that users of chat-ui would get the latest feature without having to change their `.env.local` bu it's still toggleable if users don't want it. wdyt?

---

I would keep it as simple as possible by default (so disabled).

---

Handled in 31a4941adee529bfd1a9fc8747d8af21b7797a8f